src/qml/components/PeersIndicator.qml

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// The PeersIndicator component.
+// See reference design: https://bitcoindesign.github.io/Bitcoin-Core-App/assets/images/block-clock-connection-states-big.png
+
+import QtQuick 2.15
+import "../controls"
+
+Item {
```

Comment:
b0b36d6668bcf2d14438b19d849de650951ac02f

This can be `Row`? Then can drop `width` and `height` bindings below.

Otherwise below should bind `implicitWidth` and `implicitHeight`.

Reply:
> This can be `Row`?

Yes, it can. But using `Item` provides better encapsulation and hiding implementation details. For example, `PeersIndicator { padding: 42 }` will fail in the current branch.

Reply:
> Otherwise below should bind `implicitWidth` and `implicitHeight`.

Thanks! [Updated](https://github.com/bitcoin-core/gui-qml/pull/127#issuecomment-1146592288).